.github/workflows/ci-app.yml

```diff
@@ -1,28 +1,79 @@
-name: NextJS CI
+name: React CI
```

Comment:
Open to it. It's slightly annoying when the Next.js and Flask templates are together in a monorepo, because then multiple systems are called "App" in so many spots. Is the expectation that project teams go through and rename files and directories when they're doing a monorepo?

Reply:
Updated to `CI - App`

Reply:
yeah project teams would need to rename (they have to anyways since the `/app` folder conflicts.

hadn't yet figured out the developer experience for this, shawn vanderjagt and i thought we could kick the can down the road a bit — we could maybe have the install script ask the project what they want to name the app, and then rename files/directories to the name of the app, defaulting to "app" if they don't pick a name. thoughts?